tutorial/README.md

```diff
@@ -20,10 +20,14 @@ After completing the getting started section you should be connected to an AKS c
 Clone this repository locally using whatever technique you prefer. Example:
 
 ```sh
-git clone git@github.com/azure/azure-oam-solution.git
+git clone git@github.com:Azure/azure-oam-solution.git
 ```
 
-This repository contains code you will use during the tutorial as well as a solution you can use to check your work. You might find it convenient to use the `./tutorial/flight-tracker` directory as your working directory for the bulk of the tutorial.
```

Comment:
This should still be here.

Reply:
All the tutorial works from the `tutorial` folder.

Reply:
The explanation of why we asked to clone the repo should still be here.

Reply:
This is not an explanation of why to close the repo. It is just telling the user which directory he might find convenient to use. Now, this tutorial already handles that in the commands. Moving to a different directory can break some commands.

Reply:
Ok fair point. Agreed on taking this out.